packages/jellyfish-transaction/src/script/dftx/dftx_masternode.ts

```diff
@@ -1,30 +1,42 @@
+import { SmartBuffer } from 'smart-buffer'
 import { BufferComposer, ComposableBuffer } from '../../buffer/buffer_composer'
 
 // Disabling no-return-assign makes the code cleaner with the setter and getter */
 /* eslint-disable no-return-assign */
 
 /**
- * CreateMasterNode DeFi Transaction
+ * CreateMasternode DeFi Transaction
  */
-export interface CreateMasterNode {
-  type: number // ----------------------------------| 1 byte, if 0x01 p2pkh, otherwise = p2wpkh
-  collateralPubKeyHash: string // ------------------| VarUInt{20 bytes}
-  operatorPubKeyHash?: string // -------------------| VarUInt{20 bytes}, p2pkh only, optional, use collateral pkh as default
+export interface CreateMasternode {
+  operatorType: number // --------------------------| 1 byte, 0x01 = p2pkh, 0x04 = p2wpkh
+  operatorAuthAddress: string // -------------------| VarUInt{20 bytes}
```

Comment:
Maybe we should keep the name `operatorPubKeyHash` for this one since it's technically the pubkey not the address (I just confirmed)